src/test/java/tests/DownloadAppFunctionalityTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import com.codeborne.selenide.WebDriverRunner;
+import common.TestInit;
+import io.qameta.allure.Description;
+import org.testng.annotations.Test;
+import pages.HomePage;
+
+import static com.codeborne.selenide.Selenide.closeWindow;
+import static com.codeborne.selenide.Selenide.switchTo;
+
+public class DownloadAppFunctionalityTest extends TestInit {
+
+    @Test(description = "Positive test - Check 'Download App' Button ")
+    @Description("SF2-14")
+    public void downloadAppTest(){
+        HomePage homePage = new HomePage();
```

Comment:
move it in class not in method, so you could reuse it in the other test in future